ui/lib/pki/addon/decorators/check-config.js

```diff
@@ -21,6 +22,7 @@ export function withConfig() {
       return SuperClass;
     }
     return class CheckConfig extends SuperClass {
+      @service secretMountPath;
```

Comment:
this okay to do here?

Reply:
Yes I think that is good, otherwise you rely on the parent class to inject the service.

Reply:
yep! 

and exactly @zofskeez - it seems like if this is where it's called it made sense to have